### PR TITLE
Check for use of idHelper

### PR DESCRIPTION
**Pull request by jtwebb**
Checks for use of idHelper (for repositories) with use of `insert` and `insertAndReturnFirstRow`.

### CODE REVIEW

**Review by jeffcousins — APPROVED** [review]

File: lib/rules/id-helper.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      }
+    ],
+    messages: {
+      useIdHelper: 'Use idHelper for `insert`, `insertAndReturnFirstRow` and `bulkCreate`'
+    }
+  },
+
+  create: function (context) {
+    return {
+      'CallExpression[callee.property][callee.object.name!="dbBatch"][callee.property.name=/(insert|insertAndReturnFirstRow)/]'(
```
Comment (chrisvariety) [inline lib/rules/id-helper.js]: I'm a little confused about the inclusion of `insertAndReturnFirstRow` - aren't calls to that function already wrapped via https://github.com/thoughtindustries/ti/commit/8274cb8dc615103b10dd54b23fcdee0f97ecbf13#diff-e022be96c8d8394b38b4498114410b7e0a5af8308d279f928bf6639e9e4eb5d3R96 ?

Comment (jtwebb) [reply lib/rules/id-helper.js]: Oh, you're right. Let me remove that!

Comment (jtwebb) [reply lib/rules/id-helper.js]: Removed!

File: lib/rules/id-helper.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    schema: [
+      {
+        type: 'object',
+        properties: {},
+        additionalProperties: false
+      }
+    ],
+    messages: {
+      useIdHelper: 'Use idHelper for `insert`, `insertAndReturnFirstRow` and `bulkCreate`'
```
Comment (chrisvariety) [inline lib/rules/id-helper.js]: Is this accurate still?

Comment (jtwebb) [reply lib/rules/id-helper.js]: Updated the message

**Review by chrisvariety — APPROVED** [review]

**Pull request merged**